vizro-ai/docs/pages/user-guides/retrieve-dashboard-code.md

```diff
@@ -116,16 +116,13 @@ This triggers the dashboard building process. Once Vizro-AI finishes the dashboa
         dashboard = vm.Dashboard(
            pages=[
                vm.Page(
-                   id="Tips Data Visualization",
```

Comment:
These IDs are part of Vizro-AI output and I'm not sure we should alter the output of VizroAI? wdyt?

Reply:
I can revert this if you think it's best to leave it.

Reply:
Do we need to consider if Vizro-AI output needs tweaking to remove unnecessary id values? @nadijagraca @huong-li-nguyen @maxschulz-COL

Reply:
Yes we should leave the automatically created IDs in - I think it is just more realistic to showcase a real output.

It is almost impossible to tweak vizro-ai to not have "unnecessary" IDs. The reason for that is that work on the basis of our pydantic models, and the model needs to decide itself whether it wants to fill in the optional id field.

We could enhance vizro-ai with our documentation (RAG) or even fine tune and try to teach it not to do so, but that would be a large overhead for not so much of a big problem. We could also try to figure out whether the id is necessary after the facts, ie when we export our pydantic model to python (`_to_python` of the base model), but this would be probably quite tricky, as we would need to check if any controls/other parts need it.

So overall I think this is not easy, but I also think it really isn't so bad, it provides good code that I would not be opposed to even as a dev. Yes it is not the shortest most concise, but it's also nice for if you want to add controls later, or if you want to understand your debug mode better. I don't know, I just think this is ok no?

Reply:
Ah, I didn't realise it would be so difficult to change that! No worries, let's just leave it as is then👍.

For me, it's less about having concise/short code, but rather clearly demonstrating what's needed to generate the output. In most of our examples, the `id` isn't required. I typically don't add an `id` unless I know I'll need it for something like a filter interaction. 

However, I don't mind keeping it here if it's difficult to remove. It's definitely not worth lots of work. It's not a big deal, as the code is still correct :)